### PR TITLE
Import from Lionweb models not exported from Kolasu

### LINKED ISSUE #66: Issue with LionWeb interoperability
At this time we consider only features providing AST nodes to be containments.
However AST nodes are a subset of all possible LionWeb Nodes, and if we have a feature returning something that is a LionWeb Node but not also an AST node that is treated as an attribute and not a containment.
So far it was not an issue because we only used AST nodes. However now we are using simple LionWeb Nodes in CIS and encountered this issue.

We (Martin and myself) think that the issue is here:

https://github.com/Strumenta/tylasu/blob/db40528ac077b1e25ea06d86d09db5bc22ee3b2e/src/interop/lionweb.ts#L88

We should check for something to be just a Node. Changing the code into:

```
function isLionWebNode(classifier: Classifier) {
    return classifier instanceof Concept;
}
```

Our issue was solved

### PR DESCRIPTION
**Pull request by alessiostalla**
This fixes #66

### CODE REVIEW

File: src/interop/lionweb.ts [hunk 1]
```diff
@@ -77,27 +77,39 @@ export class LanguageMapping {
...
     if (feature instanceof Containment) {
-        if (feature.type && isASTNode(feature.type)) {
+        if (feature.type && feature.type instanceof Concept && !isSpecialConcept(feature.type)) {
             def.child = true;
             def.multiple = feature.multiple;
         } else {
-            // TODO we assume that:
-            //  1) we're importing a StarLasu AST
-            //  2) containments in a StarLasu AST are either AST nodes or internal StarLasu objects like the position
+            // TODO is it possible in Lionweb to have a Containment that's not a concept?
```
Comment (ftomassetti) [inline src/interop/lionweb.ts]: Yes: it could be a ConceptInterface

**Comment by ftomassetti** [comment]
I am not sure I understand the code, but I cannot find anything obviously wrong with it.

Maybe adding a test would make the resolution more obvious, but I would not block this PR because of that.

**Review by ftomassetti — APPROVED** [review]

File: src/interop/lionweb.ts (hunk 1 above)
Comment (alessiostalla) [reply src/interop/lionweb.ts]: Ok, it makes sense to support that as well.

**Comment by alessiostalla** [comment]
I've kept the existing tests and they still pass. I've manually modified the FS language so that no concept inherits from ASTNode, that should be sufficient to verify we can actually handle those languages.

**Pull request merged**